contracts/src/v0.8/functions/dev/v1_X/FunctionsRouter.sol

```diff
@@ -377,8 +377,7 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
       commitment.estimatedTotalCostJuels,
       commitment.client,
       commitment.adminFee,
-      juelsPerGas,
-      SafeCast.toUint96(result.gasUsed),
+      juelsPerGas * SafeCast.toUint96(result.gasUsed),
```

Comment:
Making a Router change would mean deploying two different Routers, since existing networks won't be receiving a Router update for the next scheduled deployment

Reply:
Yeah called that out in the doc, probably ok to backlog until you'd be doing a v2